libs/SmartStore/SmartStore/Classes/SFSmartStore.h

```diff
@@ -360,7 +360,7 @@ NS_SWIFT_NAME(SmartStore)
  
  @return The array of updated entries in the soup.
  */
-- (NSArray<NSDictionary*>*)upsertEntries:(NSArray<NSDictionary*>*)entries toSoup:(NSString*)soupName NS_SWIFT_NAME(upsert(entries:soupName:));
+- (NSArray<NSDictionary*>*)upsertEntries:(NSArray<NSDictionary*>*)entries toSoup:(NSString*)soupName NS_SWIFT_NAME(upsert(entries:forSoupNamed:));
```

Comment:
Should we make this one unavailable in swift and add an upsertEntries that take a `NSError**` (and therefore that throws in Swift)?
(The upsert with external id path takes a `NSError** `/ throws).